configure.py

```diff
@@ -319,6 +325,24 @@ def get_var(environ_cp,
     question += ' [y/N]: '
 
   var = environ_cp.get(var_name)
+  if var is not None:
+    _var_content = var.strip().lower()
```

Comment:
Google Python style doesn't require an underscore in the beginning of a local variable name. See:
https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming

(although it requires an underscore at the beginning of a private function name).